src/ui-lib/sass/05-components/03-organisms/_job-list.scss

```diff
@@ -4,30 +4,44 @@
   }
 
   @media all and (min-width: gravy-breakpoint(regular)) {
-    display: grid;
-    grid-gap: $grav-sp-m;
-    grid-template-columns: 30% 1fr;
+    display: flex;
+    align-items: baseline;
   }
 
-  > header > p {
-    @include typi(small-text);
-    margin-top: 0;
+  > header {
+    width: 30%;
```

Comment:
The width of the `header` isn't rendering as a consistent width which means things aren't aligned as well as they could be.

![image](https://user-images.githubusercontent.com/4202934/38727727-c3751ed4-3f05-11e8-87cc-555e15970201.png)